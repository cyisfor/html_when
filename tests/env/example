foo=1
something=silly
colonel=very serious