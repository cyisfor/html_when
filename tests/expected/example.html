<!DOCTYPE html>
<html><body>This can be whatever the fuck you want.

when not censored, If you just want to omit something the assholes would censor, you can do that too.

<a href="http://[derp]/prer">derp</a>

when not censored, This syntax isn’t especially smart, so don’t get fancy with it.

What did I fucking say?

when not not censored, 

XML sucks, by the way. Don’t assign values just to make attributes exist.

something is silly!
something is very silly!
The Colonel isn’t silly, but is very serious instead.

The environment variable “foo” has been set to 1.

</body></html>
